### ISSUE TITLE
[1.16.4] Immersive Enginnering & OptiFine Problem

### ISSUE THREAD

**Issue by trevorreynen** [BluSunrize/ImmersiveEngineering#4554]
I had a problem with this mod and found out half of the problem was OptiFine. So I tested with both, just Immersive Engineering, and then OptiFine. I do have other mods, but you'll understand why I bother removing those (even though it's possible they can be causing the problem some how)

My Testing Results:
1. When **Immersive Engineering AND OptiFine** are both installed:
  - There is a problem where I can see certain parts of caves when I look through the _surface_ of water.
  - Also causes an "OpenGL Error: 1282 (Invalid operation)" message that spams every 5/10 seconds. (Most likely from OptiFine)
2. When **just Immersive Engineering** is installed:
  - There is a problem where I can see certain parts of caves when I look through the _surface_ of water.
3. When **just OptiFine** is installed:
  - No problems or errors.

Full List of Mods:
- Forge (35.1.29)
- OptiFine (HD_U_G5)
- Immersive Engineering (4.1.2-129)
- Applied Energistics 2 (8.1.0)
- Industrial Foregoing (3.2.7-7d95990)
- Titanium (3.2.4)
- Mekanism (10.0.18.445)
- Mekanism Additions (10.0.18.445)
- Mekanism Generators (10.0.18.445)
- Mekanism Tools (10.0.18.445)
I know the "instructions" said to not say the latest version, but I downloaded these mods today, 01/03/2021, so they are all the latest versions of today.

**Comment by VariableVixen** [comment]
In reverse order: the "don't say 'latest' for version" instruction means give a version number, don't _just_ say "latest" alone because nobody knows what the actual version number is in that case. You're fine on that. However, Optifine is not supported.

**Comment by trevorreynen** [comment]
> In reverse order: the "don't say 'latest' for version" instruction means give a version number, don't _just_ say "latest" alone because nobody knows what the actual version number is in that case. You're fine on that. However, Optifine is not supported.

Yeah I was just mentioning it. Anyways, even with optifine not supported i mean i can deal with turning off OpenGL Error messages. It's the other problem I don't like which happens even when optifine is not installed. I will share a screenshot below.

In my original post I said "I can see certain parts of caves when I look through the surface of water." However, I think it's actually water/lava in caves and not just random parts of caves.
![image](https://user-images.githubusercontent.com/22898630/103492248-8515ee80-4df7-11eb-9673-e1774c4e8448.png)

**Comment by trevorreynen** [comment]
Actually I just figured out the problem. OptiFine actually doesn't seem to have a problem for me other than the fact that if I turn the Graphics setting to "Fabulous!", it causes the problem in the screenshot I sent previously. Any and all other setting changes do not have problems or conflicts as far as I've seen. I had to reinstall everything to figure out what the problem was between the two mods.
